.github/workflows/merge-checks.yml

```diff
@@ -30,3 +30,5 @@ jobs:
         uses: rharkor/caching-for-turbo@v1.5
       - name: Run qa checks
         run: pnpm merge-checks
+      - name: Check deduped deps
```

Comment:
`check-deduped-deps` is already inside `merge-checks`, so no need for this

Reply:
No, I removed it. Unfortunately, it deletes node_modules, so it has to be run afterward.

Reply:
like mentioned in the other comment, we can remove this now